material-slider/src/slider.ets

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { SliderModel, SliderType } from '@ohos/material-slider'
+
+@Component
+export struct MaterialSlider {
+  @State obj: SliderModel = new SliderModel(SliderType.Continue)
+  private onCheckChange: ((value: number, mode: SliderChangeMode) => void) = null
```

Comment:
rename the callback function variable related to your library e.g onSliderChange